File: codes/FibonacciFunc/FibonacciFunc.py
def Fibo(n):
    if n == 1:
        return 0
    elif n == 2:
        return 1
    else:
        return Fibo(n-1) + Fibo(n-2)

for i in range(1, 15):
    print(Fibo(i), end=' ')

File: codes/FibonacciFunc/FibonacciFunc.java
class Fibo {
    int Fibo(int n){
        if(n==1)
            return 0;
        else if(n==2)
            return 1;
        else
            return Fibo(n-1) + Fibo(n-2);
    }
}
public class FibonacciFunc {
    public static void main(String[] args) {
        Fibo f = new Fibo();

        for (int i = 1; i < 15; i++)
            System.out.print(f.Fibo(i)+" ");
    }
}
